### DISCUSSION TITLE
Adding shadcn component framework

### DISCUSSION THREAD

**Discussion by rafaeljigau** [rjsf-team/react-jsonschema-form#4253] [Q&A]
Hello everyone!

I would like to add https://ui.shadcn.com/ as a new component framework to work with rjsf.
What are the steps I should take to achieve that?

**Comment by nickgros** [comment]
We have some instructions [here](https://rjsf-team.github.io/react-jsonschema-form/docs/advanced-customization/custom-themes). Most theme authors start with an existing theme and modify it to use their UI framework of choice. Feel free to create a PR and we can review it.

**Comment by husayt** [comment]
@rafaeljigau you can use [this as a starting point](https://github.com/tuanphung2308/next-rjsf-form-builder). You will find some shadcn components started

**Reply by heath-freenome** [reply]
@rafaeljigau Interestingly someone just added support for `shadcn` theme in the upcoming `v6` beta. Estimated time for release is end April
